### ISSUE TITLE
SDG: 17 goals grid

### ISSUE THREAD

**Issue by mlbrgl** [owid/owid-grapher#1722]
<img width="1440" alt="Screenshot 2022-10-26 at 11 33 48" src="https://user-images.githubusercontent.com/13406362/197991366-415c467d-478a-4cc6-8793-c7c958e0f0d2.png">

- [x] create AML document structure
- [x] create component
- [x] handle colors
- [x] hover state
- [x] link
- [x] align to grid
- [ ] `won't do` responsive behaviour (blocked by grid refactor)
- [x] align to typographic styles
- [x] match spacing
- [x] wrapper / title
